src/containers/auth/forgotPassword/index.js

```diff
@@ -75,9 +75,9 @@ class ForgotPassword extends PureComponent {
     const valid = this.preValidateForm({ email });
     if (valid) {
       const response = await forgotPasswordRequest({ email });
```

Comment:
You can write like this` const { status, message } = await forgotPasswordRequest({ email });`

Reply:
Done